src/chrome/content/rules/openwall.info.xml

```diff
@@ -0,0 +1,4 @@
+<ruleset name="openwall.info">
+	<target host="openwall.info" />
+	<rule from="^http:" to="https:" />
```

Comment:
Could you please also add a redirect from `www` to `^` and documenting the mismatch?

Reply:
Done.